app/helpers/patients_helper.rb

```diff
@@ -25,6 +25,10 @@ def insurance_options
      'No insurance', 'Don\'t know', 'Other (add to notes)']
   end
 
+  def line_options
```

Comment:
I think we can probably consume an existing method for this in `lines_helper`, `lines`. (which just invokes the `LINES` constant, which comes from an env var). The instinct is totally right on though!